SUPPORT.md

```diff
@@ -1,57 +1,26 @@
 OpenSSL User Support resources
 ==============================
 
-_Under Construction; not more than a collection of text fragments yet._
+See the <https://www.openssl.org/support/contracts.html> for details on how to
+obtain commercial technical support.
```

Comment:
```suggestion
If you are seeking commercial [technical support], please consult our website.

[technical support]: https://www.openssl.org/support/contracts.html
```
_(looks better in rendered markdown)_

Reply:
_I did not place the link at the end of the document into the `<!-- Links -->` section on purpose, because then the reader of the raw markdown document might not find it immediately._

Reply:
I'm not keen on that change